crates/typst/src/eval/library.rs

```diff
@@ -77,6 +77,12 @@ pub struct LangItems {
     pub enum_item: fn(number: Option<usize>, body: Content) -> Content,
     /// An item in a term list: `/ Term: Details`.
     pub term_item: fn(term: Content, description: Content) -> Content,
+    /// The constructor for the 'rgba' color kind.
+    pub rgb_func: &'static NativeFunc,
+    /// The constructor for the 'cmyk' color kind.
+    pub cmyk_func: &'static NativeFunc,
+    /// The constructor for the 'luma' color kind.
+    pub luma_func: &'static NativeFunc,
```

Comment:
I chose this over `Func` as it felt like unnecessary wrapping. On my tests, it didn't seem to affect the `LangItems` struct's `Sync` or `Send`-ability at all, so should be fine. Feel free to change this though.

Reply:
As long as it compares correcly, it should be fine.

Reply:
Sounds great. It does seem to compare correctly (I added tests with equality and inequality), so that's nice.